bookkeeper-benchmark/src/main/java/org/apache/bookkeeper/benchmark/package-info.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+ */
+/**
+ * @TODO: Write JavaDoc comment
```

Comment:
@sijie @fpj  is this the current practice in the project ? personally I don't like very much to leave "http://" links to the issue tracker in code. Issue trackers change often, as currently we are doing the switch from JIRA to github but code will remain "forever" (or for a really long time) if no one will take care of addressing the issue as soon as possible

IMHO a better way would be to disable the rule about "javadoc must be present" and create and issue like "introduce the 'javadoc must be present' rule and add missing docs".
This way we are not going to fill code with TODOs and at the same time we will not forget about the lack of such comments

http://wiki.c2.com/?TodoCommentsConsideredHarmful